### DISCUSSION TITLE
[Feature Request]能否增加根据深色和浅色模式配置不同logo和favicon.ico

### DISCUSSION THREAD

**Discussion by lurj7988** [vuepress-theme-hope/vuepress-theme-hope#3152] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/45491061/4bbce2e6-4d7f-4482-8a23-5214eb110b34)
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/45491061/14f26dd6-1e9b-46e8-9a41-3e7f86730979)
浅色模式logo正常，深色模式直接看不到了，能否针对深色和浅色模式配置不同的logo，

**Comment by Mister-Hope** [comment, marked as answer]
For logo, we have heroImageDark for homepage and logoDark for Navbar. READ THE DOCS PLZ!

For favicon, this is not something we need to support. The tab color is always syncing with system settings, not theme dark mode. Here is a reference 【【前端】深色模式favicon图标-哔哩哔哩】 https://b23.tv/NPZ5GXy
